### PR TITLE
Custom response headers

### PR DESCRIPTION
**Pull request by c-martinez**
Implementing custom response headers as per #90 

Custom headers can be set via conf file. Response headers must be a list of header-name / header-value:

```
response_headers = [('Access-Control-Allow-Origin', 'http://example.org')]
```

### CODE REVIEW

File: wsgidav.conf [hunk 1]
```diff
@@ -45,7 +45,10 @@ ssl_private_key = "wsgidav/server/sample_bogo_server.key"
 
 # Add the MS-Author-Via Response Header to OPTIONS command to allow editing
 # with Microsoft Office (default: True)
-add_header_MS_Author_Via = True	
+add_header_MS_Author_Via = True
+
+# Add custom headers
+response_headers = [('Access-Control-Allow-Origin', 'http://example.org')]
```
Comment (mar10) [inline wsgidav.conf]: Should be commented out as example, but not enabled by default.
The file also uses double quotes throughout.

File: wsgidav.conf.sample [hunk 2]
```diff
@@ -64,6 +64,9 @@ port = 8080
 # with Microsoft Office (default: True)
 add_header_MS_Author_Via = True	
 
+# Add custom headers
+response_headers = [('Access-Control-Allow-Origin', 'http://example.org')]
```
Comment (mar10) [inline wsgidav.conf.sample]: Same as above: Should be commented out as example, but not enabled by default and use double quotes

File: wsgidav/request_server.py [hunk 3]
```diff
@@ -1490,6 +1490,11 @@ def _sendResource(self, environ, start_response, isHeadMethod):
         if res.supportEtag():
             responseHeaders.append(("ETag", '"%s"' % entitytag))
 
+        if "response_headers" in environ["wsgidav.config"]:
+            customHeaders = environ["wsgidav.config"]["response_headers"]
+            for header, value in customHeaders :
```
Comment (mar10) [inline wsgidav/request_server.py]: One superfluous space at the end

**Comment by c-martinez** [comment]
Thanks for feedback -- fixed.

**Comment by mar10** [comment]
Thanks!

**Pull request merged**